platforms/quorum/configuration/roles/setup/bootnode/tasks/main.yaml

```diff
@@ -10,40 +10,11 @@
     path: "{{ bin_install_dir }}/bootnode"
   register: bootnode_stat_result
 
-# This task checks if quorum repo directory exists
-- name: Check quorum repo dir exists
-  stat:
-    path: "{{ bin_install_dir }}/quorum"
-  register: quorum_stat_result
-
-# This task clones the quorum git repository
-- name: Clone the git repo
-  git:
-    repo: "{{ bootnode.repo }}"
-    version: master
-    force: yes
-    dest: "{{ bin_install_dir }}/quorum"
-  when: not ( quorum_stat_result.stat.exists and bootnode_stat_result.stat.exists )
-
-# This task builds the bootnode binary
-- name: Make bootnode
-  make:
-    chdir: "{{ bin_install_dir }}/quorum"
-    target: bootnode
-  when: bootnode_stat_result.stat.exists == False
-
-# This task creates the bin directory, if it doesn't exist
-- name: Create bin directory
-  file:
-    path: "{{ bin_install_dir }}"
-    state: directory
-  when: bootnode_stat_result.stat.exists == False
-
-# This task puts the bootnode binary to the bin directory created in the above task
+# This task puts the bootnode binary to the bin directory created in the above task 
```

Comment:
The task description is incorrect. In the above task, it only `checks` if the bootnode binary exist in the `bin_install_dir` path.